--- ./source/CMakeLists.txt	2022-01-10 16:51:29.000000000 +0100
+++ ./source/CMakeLists.txt	2022-01-10 16:53:15.000000000 +0100
@@ -707,7 +707,6 @@
     endif()
 endif()
 
-if(X265_LATEST_TAG)
     # convert lists of link libraries into -lstdc++ -lm etc..
     foreach(LIB ${CMAKE_CXX_IMPLICIT_LINK_LIBRARIES} ${PLATFORM_LIBS})
         if(IS_ABSOLUTE ${LIB} AND EXISTS ${LIB})
@@ -730,7 +729,6 @@
     configure_file("x265.pc.in" "x265.pc" @ONLY)
     install(FILES       "${CMAKE_CURRENT_BINARY_DIR}/x265.pc"
             DESTINATION "${LIB_INSTALL_DIR}/pkgconfig")
-endif()
 
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/cmake/cmake_uninstall.cmake.in"
                "${CMAKE_CURRENT_BINARY_DIR}/cmake/cmake_uninstall.cmake"
--- ./source/common/aarch64/asm-primitives.cpp	2021-10-31 15:41:00.000000000 +0000
+++ ./source/common/aarch64/asm-primitives.cpp	2021-10-31 15:42:43.000000000 +0000
@@ -109,10 +109,10 @@
 {
     if (cpuMask & X265_CPU_NEON)
     {
-    	  // quant
+#if !HIGH_BIT_DEPTH
+    	// quant
         p.quant = PFX(quant_neon);
         
-#if !HIGH_BIT_DEPTH
         p.pu[LUMA_4x8].satd   = PFX(pixel_satd_4x8_neon);
         p.pu[LUMA_4x16].satd  = PFX(pixel_satd_4x16_neon);
 
