diff -ruN ./source/common/aarch64/asm-primitives.cpp x265_git_working/source/common/aarch64/asm-primitives.cpp
--- ./source/common/aarch64/asm-primitives.cpp	2021-10-31 15:41:00.000000000 +0000
+++ ./source/common/aarch64/asm-primitives.cpp	2021-10-31 15:42:43.000000000 +0000
@@ -109,10 +109,10 @@
 {
     if (cpuMask & X265_CPU_NEON)
     {
-    	  // quant
+#if !HIGH_BIT_DEPTH
+    	// quant
         p.quant = PFX(quant_neon);
         
-#if !HIGH_BIT_DEPTH
         p.pu[LUMA_4x8].satd   = PFX(pixel_satd_4x8_neon);
         p.pu[LUMA_4x16].satd  = PFX(pixel_satd_4x16_neon);
 
